restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/entities/SmsMessage.java

```diff
@@ -47,13 +46,20 @@
     private final Direction direction;
     private final BigDecimal price;
     private final Currency priceUnit;
+    private final String tlvTag;
```

Comment:
Should we support array here, like TlvSet, rather than just a single key/value pair? @abdulazizali77 what do you think?